cmd/gazelle/fix-update.go

```diff
@@ -456,6 +465,10 @@ func runFixUpdate(wd string, cmd command, args []string) (err error) {
 				ruleIndex.AddRule(c, r, f)
 			}
 		}
+
+		return walk.Walk2FuncResult{
+			Err: errors.Join(errs...),
```

Comment:
Just noting that this ties us to Go 1.20+, but we do already have test coverage for the minimum supported version and it appears to be at least that high.